ztp/policygenerator/testData/GenericSourceFiles/GenericNamespace.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I think these test files should be maintained a testData directory at the same level as the policyBuilder_test.go so as to differentiate e2e tests from unit tests.

Reply:
Makes sense. Done.